packages/mux-player/src/index.ts

```diff
@@ -287,6 +291,97 @@ class MuxPlayerElement extends VideoApiElement {
     this.video?.textTracks?.addEventListener('removetrack', onTrackCountChange);
   }
 
+  #setUpCaptionsMovement() {
+    type Maybe<T> = T | null | undefined;
+    type MediaController = Element & { media: HTMLVideoElement };
+
+    const mc: Maybe<MediaController> = this.shadowRoot?.querySelector('media-controller');
+
+    let selectedTrack: TextTrack;
+    const cuesmap = new WeakMap();
+
+    // toggles activeCues for a particular track depending on whether the user is active or not
+    const toggleLines = (track: TextTrack, userInactive: boolean) => {
+      const cues = Array.from((track && track.activeCues) || []) as VTTCue[];
+
+      cues.forEach((cue) => {
+        // ignore cues that are
+        // - positioned vertically via percentage.
+        // - cues that are not at the bottom
+        //   - line is less than -5
+        //   - line is between 0 and 10
+        if (!cue.snapToLines || cue.line < -5 || (cue.line >= 0 && cue.line < 10)) {
+          return;
+        }
+
+        // if the user is active or if the player is paused, the captions should be moved up
+        if (!userInactive || this.paused) {
+          // for cues that have more than one line, we want to push the cue further up
+          const lines = cue.text.split('\n').length;
+          // start at -3 to account for thumbnails as well.
+          const setTo = -3 - lines;
+
+          // if the line is already set to -4, we don't want to update it again
+          // this can happen in the same tick on chrome and safari which fire a cuechange
+          // event when the line property is changed to a different value.
+          if (cue.line === setTo) {
+            return;
+          }
+
+          if (!cuesmap.has(cue)) {
+            cuesmap.set(cue, cue.line);
+          }
+
+          // we have to set line to 0 first due to a chrome bug https://crbug.com/1308892
+          cue.line = 0;
+          cue.line = setTo;
+        } else {
+          setTimeout(() => {
+            cue.line = cuesmap.get(cue) || 'auto';
+          }, 500);
```

Comment:
this delay is so that the captions won't be partially obscured as the control bar is transitioning into being hidden.